builder/openstack/image_config.go

```diff
@@ -22,6 +22,9 @@ type ImageConfig struct {
 	// usually a project (also called the "tenant") with whom the image is
 	// shared.
 	ImageMembers []string `mapstructure:"image_members" required:"false"`
+	// True to perform the image accept so the member can see the image in his
```

Comment:
I'd like to see a bit more detail here, including the fact that for it to work your build user needs to have special privileges. Also, make it clear here that by default the value is "false"

Reply:
Makes sense.